fiddler/datadog_checks/fiddler/check.py

```diff
@@ -163,4 +166,4 @@ def check(self, _):
         # More info at
         # https://datadoghq.dev/integrations-core/base/api/#datadog_checks.base.checks.base.AgentCheck.service_check
 
-        self.service_check("fiddler.can_connect", AgentCheck.OK)
+        self.service_check("can_connect", AgentCheck.OK)
```

Comment:
Fixed this because the `__NAMESPACE__` variable was adding an extra `fiddler.` prefix to the service check name.

Reply:
Looks good.